pages/price-feeds/index.mdx

```diff
@@ -16,9 +16,3 @@ Pyth offers two different versions of price feeds:
 
 Developers may also consider using [Benchmarks](../benchmarks) to access historical Pyth prices for both on- and off-chain use.
 These historical prices can be used for settlement or other similar applications.
```

Comment:
I think we probably still want a link to the Getting Started page somewhere on the intro page right? Maybe we put in a separate paragraph with a link before this paragraph pointing them to benchmarks?